stage1_fly/run/main.go

```diff
@@ -110,15 +110,24 @@ func init() {
 	flag.StringVar(&discardString, "local-config", common.DefaultLocalConfigDir, "Local config path")
 }
 
+func checkAndAddMps(namedVolumeMounts map[types.ACName]volumeMountTuple, mountpoints []types.MountPoint) error {
```

Comment:
can we improve the naming, i.e. simply "addMountpoints"? Since this method returns an error, checking is implied.also `mountpoints []types.MountPoint` stutters, i.e. name it `mps` instead?